scripts/run-build.sh

```diff
@@ -1,12 +1,21 @@
 #!/usr/bin/env bash
 set -ex
+HERE=$(dirname $0)
+. $HERE/common
+
+# The main build env which tests and builds below
+docker build -f build.dockerfile \
+    -t $BUILD_ENV_TAG \
+    --build-arg MONTAGU_GIT_ID=$GIT_SHA \
+    --build-arg MONTAGU_GIT_BRANCH=$GIT_BRANCH \
+    .
 
 # This is the path for teamcity agents. If running locally, pass in your own docker config location
 # i.e. /home/{user}/.docker/config.json
-docker_auth_path=${1:-/opt/teamcity-agent/.docker/config.json}
+docker_auth_path=${1:-/var/lib/buildkite-agent/.docker/config.json}
```

Comment:
Note this has broken the TC build, but that will be removed in any case once this PR is approved.